cockroachcloud/cloud-api.md

```diff
@@ -640,7 +640,7 @@ Where `{sql_username}` is the username of the deleted SQL user.
 
 ## Change a SQL user's password
 
-To change a SQL user's password send a `PUT` request to the `/v1/clusters/{cluster_id}/sql-users/{sql_username}/password` endpoint. The service account associated with the secret key must have `ADMIN` [permission](console-access-management.html#service-accounts) to change a SQL user's password.
+To change a SQL user's password send a `PUT` request to the `/v1/clusters/{cluster_id}/sql-users/{sql_username}/password` endpoint. The service account associated with the secret key must have `ADMIN` [permission](managing-access.html#manage-service-accounts) to change a SQL user's password.
```

Comment:
In the new auth model, this can be done by Cluster Admin role or Org Admin (legacy) role.